### PR TITLE
Added temporary fix for fp16 test

### PR DESCRIPTION
**Pull request by a-a-egorovich**

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins ok to test

File: tools/accuracy_checker/accuracy_checker/metrics/regression.py [hunk 1]
```diff
@@ -523,6 +523,8 @@ def calculate_distance(x_coords, y_coords, selected_points):
 
 
 def mae_differ(annotation_val, prediction_val):
+    if (annotation_val.shape[0] < prediction_val.shape[0]):
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/regression.py:526]: here can be not only arrays, but also just float values, possibly you should handle this case

File: tools/accuracy_checker/accuracy_checker/metrics/regression.py [hunk 2]
```diff
@@ -523,7 +523,7 @@ def calculate_distance(x_coords, y_coords, selected_points):
 
 
 def mae_differ(annotation_val, prediction_val):
-    if (annotation_val.shape[0] < prediction_val.shape[0]):
+    if type(annotation_val) == type(prediction_val) == np.ndarray and annotation_val.shape[0] < prediction_val.shape[0]:
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/regression.py:526]:
```suggestion
    if isinstance(annotation_val, np.array) and isinstance(prediction_val, np.array) and annotation_val.shape[0] < prediction_val.shape[0]:
```

File: tools/accuracy_checker/accuracy_checker/metrics/regression.py [hunk 3]
```diff
@@ -523,6 +523,8 @@ def calculate_distance(x_coords, y_coords, selected_points):
 
 
 def mae_differ(annotation_val, prediction_val):
+    if isinstance(annotation_val, np.array) and isinstance(prediction_val, np.array) and annotation_val.shape[0] < prediction_val.shape[0]:
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/regression.py:526]:
```suggestion
    if isinstance(annotation_val, np.ndarray) and isinstance(prediction_val, np.ndarray) and annotation_val.shape[0] < prediction_val.shape[0]:
```

File: tools/accuracy_checker/accuracy_checker/metrics/regression.py [hunk 4]
```diff
@@ -523,6 +523,8 @@ def calculate_distance(x_coords, y_coords, selected_points):
 
 
 def mae_differ(annotation_val, prediction_val):
+    if isinstance(annotation_val, np.ndarray) and isinstance(prediction_val, np.ndarray) and annotation_val.shape[0] < prediction_val.shape[0]:
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/regression.py:526]:
```suggestion
    if (
        isinstance(annotation_val, np.ndarray) and isinstance(prediction_val, np.ndarray) 
        and annotation_val.shape[0] < prediction_val.shape[0]
        ):
```

File: tools/accuracy_checker/accuracy_checker/metrics/regression.py [hunk 5]
```diff
@@ -523,6 +523,11 @@ def calculate_distance(x_coords, y_coords, selected_points):
 
 
 def mae_differ(annotation_val, prediction_val):
+    if (
+        isinstance(annotation_val, np.ndarray) and isinstance(prediction_val, np.ndarray) 
+        and annotation_val.shape[0] < prediction_val.shape[0]
+        ):
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/regression.py:529]:
```suggestion
    if (isinstance(annotation_val, np.ndarray) and isinstance(prediction_val, np.ndarray) 
    and annotation_val.shape[0] < prediction_val.shape[0]
    ):
```

File: tools/accuracy_checker/accuracy_checker/metrics/regression.py [hunk 6]
```diff
@@ -523,6 +523,10 @@ def calculate_distance(x_coords, y_coords, selected_points):
 
 
 def mae_differ(annotation_val, prediction_val):
+    if (isinstance(annotation_val, np.ndarray) and isinstance(prediction_val, np.ndarray) 
+    and annotation_val.shape[0] < prediction_val.shape[0]
+    ):
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/regression.py:528]:
```suggestion
def mae_differ(annotation_val, prediction_val):
    if (isinstance(annotation_val, np.ndarray) and isinstance(prediction_val, np.ndarray)
        and annotation_val.shape[0] < prediction_val.shape[0]
        ):
```

**Comment by vladimir-dudnik** [comment]
@andrey-anufriev do we still need this?

**Pull request closed without merging**